index.html

```diff
@@ -0,0 +1,193 @@
+                    <h3>Tacos al pastor</h3>
+                    <h4>Ingredientes</h4>
+                    <ul>
+                        <li>Carne de Pollo</li>
+                        <li>1 chile de ancho</li>
+                        <li>1 chile guajilli</li>
+                        <li>1 cebolla</li>
+                        <li>1 diente de ajo</li>
+                    </ul>
+                </figure>
+
+                <figure class="video">
+                    <iframe class="youtube" width="480" height="315" src="https://www.youtube.com/embed/eG3vS9dDeoc" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                    <h3>Pechuga de pollo rellena</h3>
+                    <h4>Ingredientes</h4>
+                    <ul>
+                        <li>4 pizcas pollo en filete</li>
+                        <li>1 pizca cebolla en julianas</li>
+                        <li>1 pizca pimiento rojo en julianas</li>
+                        <li>1 taza champiñón en rebanadas</li>
+                        <li>1 diente ajo finamente picado</li>
+                        <li>1 pizca sal</li>
+                    </ul>
+
+                </figure>
+
+                <figure class="video">
+                    <iframe class="youtube" width="480" height="315" src="https://www.youtube.com/embed/xVyMN7do-2E" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                    <h3>Panqueque de Banana y avena</h3>
+                    <h4>Ingredientes</h4>
+                    <ul>
+                        <li>1 banana madura</li>
+                        <li>2 huevos</li>
+                        <li>40 gramos de copo de avena</li>
+                        <li>Aceite</li>
+                    </ul>
+
+                </figure>
+            </div>
+
+        </div>
+
+        <div class="instagram-container" id="insta">
+
+            <div class="instagram">
+
+                <div class="instatexto">
+                    <h2>Seguinos por Instagram</h2>
+                    <p><strong>Follow</strong> y sumate a nuestros seguidores para no perderte<br>nuestras recetas semanales, y participar de increíbles premios!</p>
+                    <h4><strong>Click</strong> en el logo para seguirnos!</h4>
+                    <a href="https://www.instagram.com/holapanzasana/">
+                        <img class="logoinsta hvr-pulse-grow" src="assets/images/instagram.svg"></a>
+                    <div class="underline"> </div>
+                </div>
+
+                <div class="instafoto">
+                    <img class="fotoinsta" src="assets/images/insta1.png">
+                </div>
+            </div>
+
+        </div>
+
+        <div id="formulario">
+            <h1>&bull; Envianos tu receta &bull;</h1>
+            <div class="underline">
+            </div>
```

Comment:
can u check if this kind of blocks (the empty ones can be written this way: `<div class="underline" />` otherwise just remove the breakline between the open and the close tag: `<div class="underinline"></div>`